logstash-core/api/lib/app/pipeline/stats_command.rb

```diff
@@ -7,14 +7,9 @@ def run
     #return whatever is comming out of the snapshot event, this obvoiusly
     #need to be tailored to the right metrics for this command.
     stats = service.get(:pipeline_stats)
-    snapshot = { "events" => {} }
-    stats.each_pair do |key, value|
-      if key.start_with?("events_")
-        field = key.gsub("events_", "")
-        snapshot["events"][field] = value
-      end
-    end
-    snapshot
+    { "events" => { "in" => stats[:base][:events_in].value, "filtered" => stats[:base][:events_filtered].value } }
```

Comment:
Is there any reason we don't use symbols in the hash definition? I think we should since everything from the metric store uses a symbol for index access.